docs/getting_started.rst

```diff
@@ -50,6 +47,9 @@ Next, initialize Git LFS:
 
 Finally, add the Learning Equality repo as a remote called `upstream`. That way you can keep your local checkout updated with the most recent changes:
 
+..note::
+  You may encounter an error when adding the upstream remote. If so, then it's already been added and you can skip this step.
```

Comment:
I don't see why is this note necessary, if the remote upstream has already been added, Git will specify that:

![remote-upstream-already-exists](https://user-images.githubusercontent.com/1457929/220707820-ef2e58f9-847b-4404-9854-0a467462847b.png)